terracumber_config/tf_files/tfvars/PR-TEST-uyuni.tfvars

```diff
@@ -0,0 +1,14 @@
+############ Variable unique to uyuni ########################
+
+IMAGE                  = "opensuse154-ci-pro"
```

Comment:
I wasn't aware of this image, what makes it different?

Reply:
@jordimassaguerpla can give an explanation on this. FMPOV it simply is an adjusted image with already preinstalled packages and adjusted configuration. See https://build.opensuse.org/package/show/systemsmanagement:sumaform:images:libvirt/opensuse154-ci-pr